README.md

```diff
@@ -55,26 +55,42 @@ WC3MapTranslator is a module to convert a JSON representation of WarCraft III (.
 
 This API is a core component of [Ice Sickle](https://github.com/ChiefOfGxBxL/Ice-Sickle), the next-generation world editor. Ice Sickle stores data in a JSON format, and then generates all the necessary files to assemble a .w3x MPQ archive to build a map.
 
+WC3MapTranslator is written in `TypeScript` starting from verison `3.0.0` therefore supports build in types.
+
 ## Install
-```js
+```ts
 npm install wc3maptranslator
 ```
+**Requires Node >= 8**
 
-## Usage
-```js
+## Usage JS (version < 3.0.0)
+```ts
 var Translator = require('wc3maptranslator'),
     mapJson = { // Refer to "Specification"
         units: [...],
         doodads: [...],
         ...
     };
+var unitResult = new Translator.Units.jsonToWar(mapJson.units);
+```
+## Usage TS (version >= 3.0.0)
+```ts
+// JS import
+const TranslatorLib = require('wc3maptranslator');
+const Translator = new TranslatorLib.default();
+console.log(Translator);
+
+//... or TS import
+import * as TranslatorLib from 'wc3maptranslator';
+const Translator = new TranslatorLib.default();
```

Comment:
(Correct me if I'm wrong) Does `.default()` have to be used for the TS? Isn't it possible to do:
```
import Units from 'wc3maptranslator'
const unitResult = new Units.jsonToWar(mapJson.units)
```
If it can be simplified, can the `.default()` call be gotten rid of? It seems less direct and easy to forget when choosing a Translator.

Reply:
I tried to remove the default and import it directly. But then somehow and I do not understand why the import does not work anymore. 

Maybe because of the „new“ keyword instantiations? When I have time I will look into this.

Reply:
TypeScript does not like exports on root level... I tried many different solutions but there was only one working and that one would loose the build in types. So I reverted this again. The new readme shows how to instantiate a Translator.

I removed the default (which would instantiate a Translator) since it might be confusing.